src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -141,8 +145,7 @@ def __init__(self, streams, stage, area, overlap, settings_obs=None, log_path=No
 
             # used in re-focusing method
             self._focus_points = numpy.array(focus_points) if focus_points else None
-            # triangulate focus points
-            self._tri_focus_points = Delaunay(self._focus_points[:, :2]) if focus_points else None
+            self._tri_focus_points = Delaunay(self._focus_points[:, :2]) if focus_points else None # triangulate focus points
```

Comment:
I think this is more readable with the comment above the line, why did you move it?